Source/FluentSpreadsheets/Models/Components/IndexRange.cs

```diff
@@ -9,9 +9,9 @@ namespace FluentSpreadsheets;
 {
     public IndexRange(Index start, Index end)
     {
-        if (start.Row <= end.Row || start.Column <= end.Column)
+        if (start.Row > end.Row || start.Column > end.Column)
         {
-            const string message = "IndexRange start must precede end.";
+            const string message = "IndexRange start shouldn't be greater, than the end.";
```

Comment:
Sheet indices is not really an object that has a comparative relation defined (how'd you determine if one index is greater than the other?). 
So I'd go with something like "precede or equal", as there is precedence in context of a range.

Reply:
Understandable